Makefile

```diff
@@ -103,7 +103,7 @@ test-flow-update-whitelist:
 bootstrap-test262:
 	rm -rf ./build/test262
 	mkdir -p ./build
-	git clone --branch=master --single-branch --shallow-since=2018-11-01 https://github.com/tc39/test262.git ./build/test262
```

Comment:
Whats the reason to change the date? Now it is pretty much a non shallow clone i think

Reply:
I changed it because I thought that it would clone less things :thinking:

Before it would have downloaded every commit since November, now only since January. Unless I misunderstood what shallow-since does.